src/Components/Photo_Of_The_Week_Block_Controller.php

```diff
@@ -37,8 +50,8 @@ public function get_photo(): ?array {
 			'id'       => $photo,
 			'image'    => $this->get_photo_image( $photo ),
 			'download' => get_the_permalink( $photo ),
```

Comment:
@MlKilderkin , this no longer downloads the image on-click. I think there used to be something to trigger the download when you visited the single route, but I also think I saw that got removed somewhere along the way. At any rate, clicking the download link now just takes you to the Photos of the Week archive. Would you fix this up to either auto-trigger the image download or just point to the image src directly?

Reply:
Now any permalink for single photo directs to archive. We don't do 1 click download in favor of original request make default browser behaviour. Most browsers open image/document in new tab. This also conflicts with media file download on single post. So all of this changed